context/app/static/js/shared-styles/alerts/LoginAlert.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Button from '@mui/material/Button';
+import { Alert } from './Alert';
+
+interface LoginAlertProps {
+  featureName: string;
+}
+export default function LoginAlert({ featureName }: LoginAlertProps) {
```

Comment:
Tiffany indicated that I should reuse the workspaces login alert - I figured this would be an easy way to make it reusable in other contexts.